src/ripple/rpc/impl/TransactionSign.cpp

```diff
@@ -711,22 +711,23 @@ Json::Value checkFee (
     }
 
     // Default fee in fee units.
-    std::uint64_t const feeDefault = config.TRANSACTION_FEE_BASE;
+    FeeUnit32 const feeDefault = config.TRANSACTION_FEE_BASE;
```

Comment:
I intentionally changed this from 64 to 32 to match the ledger, and to make downstream conversions easier